ui/shared/utils/constants.js

```diff
@@ -997,16 +1006,42 @@ export const getPermissionedHgmdClass = (variant, user, familiesByGuid, projectB
     familyGuid => projectByGuid[familiesByGuid[familyGuid].projectGuid].enableHgmd,
   )) && variant.hgmd && variant.hgmd.class
 
+export const clinvarSignificance = (clinvar) => {
+  let { pathogenicity, assertions } = clinvar || {}
+  if (!pathogenicity) {
+    [pathogenicity, ...assertions] = (clinvar.clinicalSignificance || '').split(/[,|]/)
+    if (pathogenicity === 'Pathogenic/Likely_pathogenic/Pathogenic') {
+      pathogenicity = 'Pathogenic/Likely_pathogenic'
+    }
+    if (!(pathogenicity.replace(' ', '_').toLowerCase() in CLINVAR_PATHOGENICITIES)) {
+      assertions = [pathogenicity, ...assertions]
+      pathogenicity = CLINVAR_DEFAULT_PATHOGENICITY
+    }
+    assertions = assertions.map(a => a.replace(/^_/, ''))
+  }
+
+  return { pathogenicity, assertions, severity: CLINVAR_PATHOGENICITIES[pathogenicity.replace(' ', '_').toLowerCase()] }
```

Comment:
If there is a `pathogenicity` the key will always be in `CLINVAR_PATHOGENICITIES` because if it isn't we assign the `CLINVAR_DEFAULT_PATHOGENICITY` above, so theres no need to duplicate that here. If there is no `pathogenicity` then `severity` will be `undefined`, which is the desired behavior